.resources/strings.json

```diff
@@ -19,130 +20,148 @@
 		"fr": "Fichiers sélectionnés",
 		"pl": "Wybrane pliki",
 		"ja": "選択されたファイル",
-		"es": "Archivos seleccionados"
+		"es": "Archivos seleccionados",
+		"de": "Ausgewählte Dateien"
 	},
 	"table.header.exif-before": {
 		"en": "# Exif Before",
 		"fr": "# Exif Avant",
 		"pl": "# Exif przed",
 		"ja": "# 処理前の Exif",
-		"es": "Exif antes"
+		"es": "Exif antes",
+		"de": "# Exif bevor"
 	},
 	"table.header.exif-after": {
 		"en": "# Exif After",
 		"fr": "# Exif Après",
 		"pl": "# Exif po",
 		"ja": "# 処理後の Exif",
-		"es": "Exif después"
+		"es": "Exif después",
+		"de": "# Exif danach"
```

Comment:
besser: # Exif nachher

Reply:
> besser: # Exif nachher

thank you, need PR is created
https://github.com/szTheory/exifcleaner/pull/88